docs/notebooks/word2vec.ipynb

```diff
@@ -1295,6 +1295,34 @@
     "print(train_times_table)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Adding Word2Vec \"model to dict\" method to production pipeline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def model_to_dict():\n",
```

Comment:
You no need this function for oneliner in notebook, 
`most_similars_precalc = {word : model.wv.most_similar(word) for word in model.wv.index2word}` enough here